### PR TITLE
Adding Netlify Badge in ReadME.md

### PR DESCRIPTION
**Pull request by vishakhanihore**
@helfi92 @djmitche Please check

### CODE REVIEW

File: changelog/netlify_badge.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+level: silent
+reference: netlify badge
+---
```
Comment (helfi92) [inline changelog/netlify_badge.md]: Delete this file then from the root repo run:

```
yarn changelog --silent --no-bug
```

File: README.md [hunk 2]
```diff
@@ -22,6 +22,9 @@
   <a href="https://codecov.io/gh/taskcluster/taskcluster">
     <img src="https://codecov.io/gh/taskcluster/taskcluster/branch/master/graph/badge.svg" />
   </a>
+  <a href="https://app.netlify.com/sites/taskcluster-web/deploys">
```
Comment (helfi92) [inline README.md]: Is the link different from what's in https://github.com/taskcluster/taskcluster/pull/2032#issuecomment-555685861?

**Comment by vishakhanihore** [comment]
The badge is different from the netlify, Since netlify badge is of
different design and shape from the other badges.Hence,I just used the
application id from the following.

**Comment by helfi92** [comment]
> The badge is different from the netlify, Since netlify badge is of
different design and shape from the other badges.Hence,I just used the
application id from the following.

Sounds good.

**Comment by vishakhanihore** [comment]
I will check it out.

**Comment by vishakhanihore** [comment]
@helfi92 Please have a look

**Comment by vishakhanihore** [comment]
Should I squash my commits ?

**Comment by helfi92** [comment]
No need. I am able to squash before merging:

Screen Shot 2019-11-20 at 1 20 17 PM

**Comment by vishakhanihore** [comment]
Great!

**Pull request merged**